extensions/cornerstone/src/hps/frameView.ts

```diff
@@ -20,7 +20,7 @@ const frameView: Types.HangingProtocol.Protocol = {
         },
         {
           attribute: 'isDisplaySetFromUrl',
-          weight: 10,
+          weight: 20,
```

Comment:
Why is this change being included?

Reply:
The weight was the same as `numImages > 0`, which didn't make the RT or SEG appear as the starting series, so I'm giving more weight to the URL.